src/parser/shaman/enhancement/modules/core/Rockbiter.tsx

```diff
@@ -34,9 +35,7 @@ class Rockbiter extends Analyzer {
   }
 
   onEnergize(event: EnergizeEvent) {
-    if (event.classResources &&
-      event.classResources[0].amount >=
-      MAELSTROM_THRESHOLD) {
+    if (event.classResources && event.classResources[0].amount >= MAELSTROM_THRESHOLD) {
```

Comment:
Zero doesnt want people using classResources[0]

https://github.com/WoWAnalyzer/WoWAnalyzer/issues/2387

Reply:
I've changed this into a classResources.find